ts_scripts/utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+
+
+def is_gpu_instance():
+    return True if os.system("nvidia-smi") == 0 else False
+
+
+def is_conda_env():
+    return True if os.system("conda") == 0 else False
+
+
+def check_python_version():
+    req_version = (3, 6)
```

Comment:
@harshbafna @maaquib Why are we pinning to 3.6 version? We need support for python >= 3.6

Reply:
@chauhang: The minor version check is for `>=6` only in line no. #17. :-)